packages/models/src/eservice/protobufConverterToV2.ts

```diff
@@ -119,6 +119,12 @@ export const toDescriptorV2 = (input: Descriptor): EServiceDescriptorV2 => ({
   archivedAt: dateToBigInt(input.archivedAt),
   rejectionReasons:
     input.rejectionReasons?.map(toDescriptorRejectedReasonV2) ?? [],
+  templateVersionRef: input.templateVersionRef
+    ? {
+        id: input.templateVersionRef.id,
+        interfaceMetadata: input.templateVersionRef.interfaceMetadata,
+      }
+    : undefined,
```

Comment:
Leftover? This is already part of the spread `...input` above and we don't have any date or enum that needs ad-hoc conversion.

```suggestion
```

Reply:
Handled in [16390c6](https://github.com/pagopa/interop-be-monorepo/pull/1643/commits/16390c62331b41f9720acdee13851d34b92ae753)